lua/packer.lua

```diff
@@ -267,7 +267,11 @@ manage = function(plugin_data)
   plugins[plugin_spec.short_name].url = util.remove_ending_git_url(plugin_spec.url)
 
   if plugin_spec.requires and config.ensure_dependencies then
-    if type(plugin_spec.requires) == 'string' then
+    -- Handle single plugins given as strings or single plugin specs given as tables
+    if
+      type(plugin_spec.requires) == 'string'
+      or (type(plugin_spec.requires) == 'table' and #plugin_spec.requires == 1)
```

Comment:
@wbthomason how would this behave if `requires` is `requires = {{'name/thing', config = blah}}`, seems like it would wrap the plugin in another list?. Should it be if 
```suggestion
      or (type(plugin_spec.requires) == 'table' and not vim.tbl_islist(plugin_spec.requires) and #plugin_spec.requires == 1)
```

Reply:
You're right, that would be better. Thanks!